### PR TITLE
[typo](doc)delete document optimization

### PR DESCRIPTION
**Pull request by caoliang-web**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/en/docs/ecosystem/flink-doris-connector.md [hunk 1]
```diff
@@ -531,7 +531,7 @@ When Exactly-Once is not required, it can also be solved by turning off 2PC comm
 Occurred in the Commit phase, the transaction ID recorded in the checkpoint has expired on the FE side, and the above error will occur when committing again at this time.
 At this time, it cannot be started from the checkpoint, and the expiration time can be extended by modifying the streaming_label_keep_max_second configuration in fe.conf, which defaults to 12 hours.
 
-6. **errCode = 2, detailMessage = current running txns on db 10006 is 100, larger than limit 100**
+6. **errCode = 2, detailMessage = current running txns on db 10006 is 1000, larger than limit 1000**
 
 This is because the concurrent import of the same library exceeds 100, which can be solved by adjusting the parameter `max_running_txn_num_per_db` of fe.conf. For details, please refer to [max_running_txn_num_per_db](https://doris.apache.org/zh-CN/docs/dev/admin-manual/config/fe-config/#max_running_txn_num_per_db)
```
Comment (zy-kkk) [inline docs/en/docs/ecosystem/flink-doris-connector.md:536]: Do we need to change 100 to 1000 here as well？

File: docs/zh-CN/docs/ecosystem/flink-doris-connector.md [hunk 2]
```diff
@@ -528,7 +528,7 @@ Exactly-Once场景下，Flink Job重启时必须从最新的Checkpoint/Savepoint
 发生在Commit阶段，checkpoint里面记录的事务ID，在FE侧已经过期，此时再次commit就会出现上述错误。
 此时无法从checkpoint启动，后续可通过修改fe.conf的streaming_label_keep_max_second配置来延长过期时间，默认12小时。
 
-6. **errCode = 2, detailMessage = current running txns on db 10006 is 100, larger than limit 100**
+6. **errCode = 2, detailMessage = current running txns on db 10006 is 1000, larger than limit 1000**
 
 这是因为同一个库并发导入超过了100，可通过调整 fe.conf的参数 `max_running_txn_num_per_db` 来解决。具体可参考 [max_running_txn_num_per_db](https://doris.apache.org/zh-CN/docs/dev/admin-manual/config/fe-config/#max_running_txn_num_per_db)
```
Comment (zy-kkk) [inline docs/zh-CN/docs/ecosystem/flink-doris-connector.md:533]: ditto

Comment (caoliang-web) [reply docs/zh-CN/docs/ecosystem/flink-doris-connector.md]: Thank you for your point, I have edited

**Review by zy-kkk — APPROVED** [review]
LGTM

**Review by JNSimba — APPROVED** [review]
LGTM

**Pull request merged**